app/views/events/index.html.erb

```diff
@@ -51,7 +50,8 @@
   <% end %>
 <% else %>
   <br>
-  <h4>Upcoming events</h4>
+ 
+  <%= "Unfortunately we don't have any upcomming CSS EVENT currently, we will keep you posted when there's any event" if @future_events.empty? %>
```

Comment:
If you can think of some better line then please feel free to suggest. I was thinking we can also have a text box there if there's no future event that would ask anyone visiting the event page to suggest to us CSS of an event they would like CSS to have in the coming months. How do you feel about it @EricPickup

Reply:
Good idea

Reply:
```suggestion
  <%= "There are no future events scheduled at the moment. Please check back later." if @future_events.empty? %>
```

@amanpatel123 this message is good enough for now.  We'll hopefully never get to the point where there are no scheduled events so I wouldn't bother going through the work of making a suggestion box